### ISSUE TITLE
Question: Support for custom font variation settings

### ISSUE THREAD

**Issue by tallchai** [google/fonts#2840]
Hi 👋 

I wanted to know if Google Fonts supports custom `font-variation-settings` properties on the font demo page?
I am building a font with a custom variation setting for `cntr` (contrast). But I wanted to know if (when) the font is published, will the font demo page include the variation property slider for that property?

Thanks.

**Comment by davelab6** [comment]
Yes, eventually. The first thing is to register the axis definition in our repo here.
